sourcegraph/jetbrains
# Chat: add support to press up/down arrow keys to open previous messages

It fixes https://github.com/sourcegraph/sourcegraph/issues/54933

Previously, you had to copy-paste a previous message if you want to send it again with some changes. This PR fixes the problem by adding up/down arrow shortcuts to load the input field with messages from your chat history. By default, up/down work just like normal. This feature can only be activated when the input field is empty.

## Test plan
* Open new cody chat window
* Send message 1
* Send message 2
* Press ⬆️, message 2 should appear in the prompt
* Press ⬆️, message 1 should appear in the prompt
* Press ⬇️, message 2 should again appear in the prompt
* Modify message 2 and send
* Press ⬆️, modified message 2 should appear
